pkg/reconciler/route/traffic/rollout.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// rollout.go contains the types and functions to deal with
+// gradual rollout of the new revision for a configuration target.
+// The types in this file are expected to be serialized as strings
+// and used as annotations for progressive rollout logic.
+
+package traffic
+
+import "knative.dev/networking/pkg/apis/networking"
+
+// RolloutAnnotationKey is the annotation key for storing
+// the rollout state in the Annotations of the Kingress or Route.Status.
+const RolloutAnnotationKey = networking.GroupName + "/rollout"
```

Comment:
maybe better in register.go with the other keys, or is this intentionally kept separate?

Reply:
For now I'll keep it here, since maybe its scope will be only route.go (then I'll just move and privatize).

Reply:
Why not start with private and only open up on request then? :thinking: Catching public things that should be private is much harder than the other way round.

Reply:
ok. did that.